packages/tocco-ui/src/StatedValue/StatedValue.stories.js

```diff
@@ -59,8 +59,8 @@ class StatedValueWrapper extends React.Component {
           onChange={this.handleChange}
           style={{
             border: 0,
-            cursor: knobs.immutable ? 'not-allowed' : 'auto',
-            color: knobs.immutable ? '#909090' : '#000',
+            cursor: knobs.immutable && knobs.type === 'field' ? 'not-allowed' : 'auto',
```

Comment:
Chould you change the stories title so it wont have a separate chapter in storybook:
![image](https://user-images.githubusercontent.com/7300481/57600366-eb326b80-7559-11e9-8c8d-6e8086728865.png)